core/arch/arm/kernel/user_ta.c

```diff
@@ -313,6 +313,14 @@ static TEE_Result ta_load(const TEE_UUID *uuid,
 		goto error_return;
 	}
 
+	if (ta_head->flags & TA_FLAG_MULTI_SESSION &&
```

Comment:
From the GP spec, we should just ignore the unmatched flags but not refuse. Maybe a warning message is needed here.